README.md

```diff
@@ -10,18 +10,33 @@ This README will provide directions for building, testing, and debugging that co
 
 * NSM_NAME - A string value of network service client name (default "nsc")
 * NSM_CONNECT_TO - A Network service Manager connectTo URL (default "unix:///var/lib/networkservicemesh/nsm.io.sock")
-* NSM_MAX_TOKEN_LIFETIME -  A token lifetime duration (default 24h)                                                            
-* NSM_ROUTES - A comma separated list of routes.                                                                                 
+* NSM_MAX_TOKEN_LIFETIME -  A token lifetime duration (default 24h)
+* NSM_ROUTES - A comma separated list of routes.
```

Comment:
Why would a client provide ROUTES?

Reply:
Imagine we have some `a.a.a.a/24` VPN and a NSE providing such VPN service. There are some predefined ` -> b.b.b.b/24`, ` -> c.c.c.c/24` routers in this VPN, but we don't want to set such routes automatically for all clients requesting `a.a.a.a/24` VPN service.
So if client needs to be configured with routes to the needed subnets, we can simply use `NSM_ROUTES` and get `a.a.a.a/24` kernel interface with routes configured to the subnets we need.
**Important:** client itself can't setup its routes because it lacks privileges, so if we don't specify `NSM_ROUTES` and someone hacks the running client, it still can't access `b.b.b.b/24`, ... subnets and make some dirty stuff. So it is the reason why we don't want to let all clients access all subnets by default.

Reply:
Removed.